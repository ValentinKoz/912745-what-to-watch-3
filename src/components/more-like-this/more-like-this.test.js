import React from "react";
import renderer from "react-test-renderer";
import MoreLikeThis from "./more-like-this.jsx";

const films = [
  {
    genre: `action`,
    title: `Parasite`,
    releaseDate: 0,
    poster: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    bgPoster: ``,
    ratingScore: ``,
    ratingLevel: ``,
    ratingCount: 1000,
    text: ``,
    director: ``,
    starring: ``,
    video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  }, {
    genre: `adventure`,
    title: `Avengers: Endgame`,
    releaseDate: 0,
    poster: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    bgPoster: ``,
    ratingScore: ``,
    ratingLevel: ``,
    ratingCount: 1000,
    text: ``,
    director: ``,
    starring: ``,
    video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  }, {
    genre: `action`,
    title: `US`,
    releaseDate: 0,
    poster: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    bgPoster: ``,
    ratingScore: ``,
    ratingLevel: ``,
    ratingCount: 1000,
    text: ``,
    director: ``,
    starring: ``,
    video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  }, {
    genre: `adventure`,
    title: `Booksmart`,
    releaseDate: 0,
    poster: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    bgPoster: ``,
    ratingScore: ``,
    ratingLevel: ``,
    ratingCount: 1000,
    text: ``,
    director: ``,
    starring: ``,
    video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  }
];

it(`ListMovie is renderer correctly`, () => {
  const tree = renderer.create(
      <MoreLikeThis films={films} currentFilm={films[0]} currentGenre={`action`} onCardClickHandle={() => {}}/>, {
        createNodeMock: () => {
          return {};
        }
      }).toJSON();

  expect(tree).toMatchSnapshot();
});
