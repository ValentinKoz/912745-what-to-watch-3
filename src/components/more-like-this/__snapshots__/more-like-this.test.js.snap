// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListMovie is renderer correctly 1`] = `
<div
  className="page-content"
>
  <section
    className="catalog catalog--like-this"
  >
    <h2
      className="catalog__title"
    >
      More like this
    </h2>
    <div
      className="catalog__movies-list"
    >
       
    </div>
  </section>
</div>
`;
